src/interfaces/strategy.ts

```diff
@@ -179,6 +185,12 @@ export class StrategyInterface<T extends ChainId> extends ServiceInterface<T> {
   }
 
   private async fetchVaultsData(): Promise<VaultData[]> {
-    return fetch("https://d28fcsszptni1s.cloudfront.net/v1/chains/1/vaults/all").then(res => res.json());
+    const res = await fetch("https://d28fcsszptni1s.cloudfront.net/v1/chains/1/vaults/all");
```

Comment:
Need to pass the correct chain id here. E.g. https://d28fcsszptni1s.cloudfront.net/v1/chains/250/vaults/all

Reply:
Addressed with https://github.com/yearn/yearn-sdk/pull/201/commits/3b64adafe509e38d7a88c9b48a7b56e1c2e886bf

Will that cloudfront endpoint remain the same for all chains?